stl/src/dllmain.cpp

```diff
@@ -9,17 +9,15 @@
 #if defined(_CRT_APP)

```

Comment:
I have a mild preference for the simpler `#ifdef _CRT_APP` (and on 16; don't forget to update the `#endif` comments appropriately).

Reply:
Done. Also changed src/filesys.cpp which was highly systematic. (Other files test `_CRT_APP` but with less systematic patterns, so I didn't change them.)